tensorboard/webapp/runs/store/runs_reducers_test.ts

```diff
@@ -238,7 +247,67 @@ describe('runs_reducers', () => {
           ['lambda', colorUtils.CHART_COLOR_PALLETE[3]],
         ])
       );
-      expect(nextState.data.nextGroupColorIndex).toBe(4);
+    });
+
+    describe('advanced grouping', () => {
+      it('assigns default color to by experiment', () => {
+        const state = buildRunsState({
+          groupBy: {
+            key: GroupByKey.EXPERIMENT,
+          },
+          defaultRunColor: new Map([
+            ['foo', '#aaa'],
+            ['bar', '#aaa'],
```

Comment:
does this runs these two runs ('foo', 'bar') are a group (they're on the same experiment?)

Reply:
Wrote a comment; I was simulating data inconsistency.